### ISSUE TITLE
Feature request: add tile_effect and multizone_effect commands

### ISSUE THREAD

**Issue by Djelibeybi** [delfick/photons-interactor#2]
The `transform` command is able to do waveform effects, i.e. Pulse and Breathe, but can't enable firmware effects like Morph, Flame or Move. 

It would be super peachy keen if there were a set of `effects` command like there are for animations, e.g. `effect/start`, `effect/stop` and `effect/status`.

**Comment by delfick** [comment]
could do.

Just make something like https://github.com/delfick/photons-interactor/blob/master/photons_interactor/commander/commands/control.py#L68

**Comment by Djelibeybi** [comment]
I'm not sure my Python knowledge is sufficient to understand how to implement this or else I would've submitted a PR, instead of a request. :) I'll smash my head against the keyboard for a while to see if my monkey brain can work it out, though.

**Comment by delfick** [comment]
I believe in you!

You can also write tests similar to https://github.com/delfick/photons-interactor/blob/master/tests/commander/commands/test_control.py#L209

**Comment by delfick** [comment]
You also have https://github.com/delfick/photons-core/blob/master/photons_control/tile.py#L76

and https://github.com/delfick/photons-core/blob/master/photons_control/multizone.py#L257

**Comment by Djelibeybi** [comment]
I solved it via the use of `set` instead: https://omg.dje.li/2020/02/advanced-lifx-lan-control-for-flic-2-with-photons-interactor/#what-else-can-i-do

**Comment by delfick** [comment]
cool :)

**Comment by Djelibeybi** [comment]
I'm going to leave this open because there are still some things that would be more elegant, like splatting a theme onto a Beam first via the `multizone_effect` command from Photons.

**Comment by delfick** [comment]
Ok, I've made this now.

https://github.com/delfick/photons-interactor/commit/928d768c45a8d0f8bf082e1c8ae88efafc7e122d

It's released to pypi as [0.6.3](https://photons-interactor.readthedocs.io/en/latest/usage/changelog.html) and it's currently [building](https://travis-ci.org/delfick/photons-interactor/builds/659748108) now so when that's done the docker container will be made :)

usage looks like

```
./command effects/run '{"matrix_animation": "FLAME", "linear_animation": "MOVE", "apply_theme": true}'
```

and 
```
# Both of these default to true, but you can disable one or the other if you want
# it's a shortcut to specifying matrix_animation or linear_animation as OFF in the effects/run
./command effects/stop '{"stop_matrix": true, "stop_linear": true}'
```

and to get the currently running effects ``effects/status``
